Modules/Scripted/DICOM/Resources/UI/DICOM.ui

```diff
@@ -181,7 +181,7 @@
        </layout>
       </item>
       <item row="2" column="0">
-       <widget class="QLabel" name="label_7">
+       <widget class="QLabel" name="labelStartStorageListenerOnStartup">
```

Comment:
```suggestion
       <widget class="QLabel" name="runListenerAtStartLabel">
```

Since it is the label associated with `runListenerAtStart`, I suggest to have a corresponding name.

We could also rename `runListenerAtStart` to `runStorageListenerAtStartCheckBox` and then have `runStorageListenerAtStartLabel`

That said, if we rename the checkbox ... we need to make sure it is not looked up by some code (it shouldn't)